awx/ui/client/lib/components/list/row-item.partial.html

```diff
@@ -1,12 +1,11 @@
 <div class="at-RowItem" ng-class="{'at-RowItem--isHeader': headerValue, 'at-RowItem--inline': inline}"
     ng-show="status || headerValue || value || valueBindHtml || (smartStatus && smartStatus.summary_fields.recent_jobs.length) || (tagValues && tagValues.length)">
         <div class="at-RowItem-status" ng-if="status">
-            <a ng-if="headerLink" ng-href="{{ headerLink }}"
-                aw-tool-tip="{{ statusTip }}" aw-tip-watch="statusTip"
-                data-placement="top">
-                    <i class="fa icon-job-{{ status }}"></i>
+            <a ng-if="statusTip" ng-click="statusClick()">
```

Comment:
this should be an "a" tag if it doesn't link to something.  you either want div or button